app/helpers/pieces_helper.rb

```diff
@@ -0,0 +1,13 @@
+module PiecesHelper
+  def render_piece_highlight(x, y)
+    piece = @game.retrieve_piece(x, y)
+    return unless piece.present?
+    if piece == @piece
+      return content_tag(:div, image_tag(piece.icon), class: 'highlight')
+    else
+      return image_tag(piece.icon)
+    end
+    link_to(raw(content_tag(:div, ' ', class: 'clickable-square')), piece_path(@piece, x_coordinate: x, y_coordinate: y), method: 'PUT')
+    end
```

Comment:
After I took out line 9, it told me that line 10 had unreachable code.  I removed it, not sure if I was supposed to.  I can put it back in if needed.